src/Evolution/Systems/Cce/AnalyticBoundaryDataManager.hpp

```diff
@@ -108,7 +108,12 @@ void AnalyticBoundaryDataManager::write_news(
   auto observer_proxy = Parallel::get_parallel_component<
       observers::ObserverWriter<Metavariables>>(
       cache)[static_cast<size_t>(Parallel::my_node(*my_proxy.ckLocal()))];
+  std::string label_string;
```

Comment:
`const`, and probably better to write `const std::string label_string{""};` (I actually thought clang-tidy it doesn't approve of this syntax). Even better:

```cpp
const std::string prefix = generator_->noninertial_news() ? "Noninertial_" : "";
```